backend/controllers/posts-controller.js

```diff
@@ -120,7 +120,7 @@ export const updatePostHandler = async (req, res) => {
 
 export const deletePostByIdHandler = async (req, res) => {
   try {
-    const post = await Post.findByIdAndRemove(req.params.id);
+    const post = await Post.findByIdAndDelete(req.params.id);
 
```

Comment:
It is recommended that we use findByIdAndDelete than findByIdAndRemove as the latter is deprecated, so updated.

https://mongoosejs.com/docs/migrating_to_8.html

<img width="844" alt="image" src="https://github.com/krishnaacharyaa/wanderlust/assets/116620586/4d18eacb-47da-403c-9d3c-a894e164fc76">

Reply:
Hey @krishnaacharyaa good man, you have mentioned the reason behind change.
Suggestion: From next time mention these details over PR description.

Reply:
> From next time mention these details over PR description.
The reason behind commenting over the specific lines was to make the life of reviewer easy tbh. But if it is standard that we mention everything in the PR description, will do from the next time.

Reply:
I believe there is no hard template, 
but I found details over the description to be handy.

Let's try out this over few more PRs.